dsgrid/config/project_config.py

```diff
@@ -613,6 +613,48 @@ def _iter_datasets_by_status(self, status: DatasetRegistryStatus) -> InputDatase
             if dataset.status == status:
                 yield dataset
 
+    @track_timing(timer_stats_collector)
+    def make_dimension_association_table(self, data_source=None):
+        """Return a table with associations across all dimensions.
+
+        Parameters
+        ----------
+        data_source : DimensionType, optional
+            Filter table by data_source, by default None.
+
+        Returns
+        -------
+        pyspark.sql.DataFrame
+
+        """
+        ds = DimensionType.DATA_SOURCE.value
+        table = self.dimension_associations.table
```

Comment:
I have trouble visualizing this

Reply:
I think that we decided that since we can't guarantee anything, it's up to the project coordinator to specify the dimension order in the project config.